### DISCUSSION TITLE
Export all config to zip/upload to cloud

### DISCUSSION THREAD

**Discussion by BenColeAu** [obsproject/obs-studio#10638] [Development Questions] [answered]
Dropping an idea here that would be helpful for those that host events and add portability for streamers.
Current state OBS has separate functions to export profiles, scenes etc..
This idea would create a single function to export all config in OBS to a single ZIP file with user data such as images copied into the ZIP.
This could later be expanded to uploading to a cloud storage location Google Drive, Onedrive etc...

**Comment by sebastian-s-beckmann** [comment, marked as answer]
An export with assets - in one form or another - is something we’d like to have in the future, but there is no ETA and still many things to be worked out.
In the future, make sure to put ideas on our [ideas page](https://ideas.obsproject.com/), not GitHub Discussions. Although I’m pretty sure such a request exists there already.
